src/components/card/card.stories.tsx

```diff
@@ -0,0 +1,339 @@
+import React from 'react'
+
+import { CardGroup } from './CardGroup/CardGroup'
+import { Card } from './Card/Card'
+import { Button } from '../Button/Button'
+
+export default {
+  title: 'Card',
+  parameters: {
+    info: `
+USWDS 2.0 Card components
+
+Source: https://designsystem.digital.gov/components/card/
+`,
+  },
+}
+
+const card = (
+  <Card
+    className="tablet:grid-col-4"
+    standardLayout="default"
+    header={
+      <header className="usa-card__header">
+        <h2 className="usa-card__heading">Card</h2>
+      </header>
+    }
+    footer={
+      <div className="usa-card__footer">
+        <Button type="button" className="usa-button">
+          Example Button
+        </Button>
+      </div>
+    }>
+    <div className="usa-card__body">
+      <p> This is a standard card with a button in the footer. </p>
+    </div>
+  </Card>
+)
+
+const cardWithMedia = (
+  <Card
+    className="tablet:grid-col-4"
+    standardLayout="default"
+    header={
+      <header className="usa-card__header">
+        <h2 className="usa-card__heading">Card With Media</h2>
+      </header>
+    }
+    footer={
+      <div className="usa-card__footer">
+        <Button type="button" className="usa-button">
+          Example Button
+        </Button>
+      </div>
+    }
+    media={
+      <div className="usa-card__media">
+        <div className="usa-card__img">
+          {/* Are we allowed to use this image? */}
```

Comment:
it looks like https://unsplash.com/ is for free images, so sounds fine with me! I vote for let's not add it to the repo.